src/org/zaproxy/zap/extension/tlsdebug/resources/Messages.properties

```diff
@@ -0,0 +1,16 @@
+# ZAP Script Console 
+#
+# This file defines the default (English) variants of all of the internationalised messages
+
+tlsdebug.desc							= TLS Debug panel 
+tlsdebug.panel.title					= TLS Debug
+tlsdebug.panel.mnemonic				= k

```

Comment:
Also don't worry about the tabbing/spacing in the properties file, it ends up being a headache to maintain. You can just use `key=value`.